internal/hooks/NewRound_IdleMobs.go

```diff
@@ -23,6 +24,8 @@ func IdleMobs(e events.Event) events.ListenerReturn {
 
 	// evt := e.(events.NewRound)
 
+	mobPathAnnounce := false // useful for debugging purposes.
```

Comment:
Do we want this to be toggled in-game at or or is it just a short term debug flag?

Reply:
It's just a way to test in game - the mob will announce what it's doing.

At some point a number of special debug flags should probably be defined for this sort of thing... maybe in the config file, since we can use `server set` to change their values on/off.

I just got tired of commenting it out/in so wrapped it in a boolean for now. Not ideal, but useful for now while we work out any kinks.